src/tools/perf/lib/libperf_int.h

```diff
@@ -44,6 +44,8 @@ typedef struct ucx_perf_thread_context ucx_perf_thread_context_t;
 
 
 typedef ucs_status_t (*ucx_perf_init_func_t)(ucx_perf_context_t *perf);
+#define ucx_perf_init_func_empty \
```

Comment:
IMO we don't need to define these macros and use casting directly, also it looks weird that some fields have casting and other don't have

Reply:
We can refactor ather fields in next PR with this approach it it's accepted, IMO casting every field looks a bit ugly